components/Tag.js

```diff
@@ -1,17 +1,33 @@
 import Link from "../components/Link";
 import { abbreviateNumber } from "../services/utils/abbreviateNumbers";
 
-export default function Tag({ name, total }) {
+export default function Tag({ name, total, path, currentInput, selected, method }) {
+
   return (
     <div className="relative">
       <Link
-        href={`/search?search=${name}`}
-        className="flex flex-row p-1 m-2 rounded-lg text-sm font-mono border-2 cursor-pointer shadow-none hover:border-orange-600"
+        onClick={()=>{
```

Comment:
I added ...

```js
          if (!currentInput) {
            return;
          }
```

and I think it fixed it, let me push to a branch @Ananya2001-an and you can review if you have time

Reply:
Yes sure I will check

Reply:
I pushed to here https://github.com/EddieHubCommunity/LinkFree/pull/4375